src/modules/FavoritesPage/FavoritesPage.module.scss

```diff
@@ -0,0 +1,95 @@
+@import '../../styles/utils/variables/colors';
+@import '../../styles/utils/mixins/mixin-media';
+@import '../../styles/blocks/grid';
```

Comment:
I guess you can combine all utils imports in 1 file, to import only that 1 file, not everything ;)